Tests/ValidationTests.swift

```diff
@@ -247,6 +247,78 @@ final class ContentTypeValidationTestCase: BaseTestCase {
         }
     }
 
+    func testThatValidationForRequestWithUnacceptableContentTypeResponseFailsWhenMultipleContentTypesAreAcceptable() {
+        // Given
+        let endpoint = Endpoint.xml
+
+        let expectation1 = expectation(description: "request should succeed and return xml")
```

Comment:
These can be given better names. Perhaps `request1DidComplete` and `request2DidComplete`? And update the `description`s to indicate which request it's talking about.